sync-repos.sh

```diff
@@ -7,6 +7,7 @@ GENIENLP_HEAD=`git rev-parse HEAD`
 if [ -n "${GENIENLP_VERSION}" ] && [ "${GENIENLP_VERSION}" != "${GENIENLP_HEAD}" ]; then
   git fetch
   git checkout ${GENIENLP_VERSION}
+  pip3 uninstall bootleg -y
```

Comment:
We should rebuild the base image when we switch to bootleg 1.0.0 instead of doing this.

Reply:
Yeah, but I still need this since I'm working on newer versions of bootleg so have to install from git hashes still. After we update to bootleg 1.0.2 (next release) we can remove this.